ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml

```diff
@@ -183,6 +183,11 @@ tests:
   steps:
     cluster_profile: gcp-openshift-gce-devel-ci-2
     workflow: openshift-e2e-gcp
+- as: e2e-gcp-techpreview
+  interval: 48h
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    workflow: openshift-e2e-gcp-techpreview
```

Comment:
We added configs for 4.10, can you add these there as well?

- ci-operator/config/openshift/release/openshift-release-master__ci-4.10.yaml
- ci-operator/config/openshift/release/openshift-release-master__nightly-4.10.yaml